src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -31,7 +31,7 @@
 /**
  * <p>
  * Restricts the number of return statements in methods, constructors and lambda expressions
- * (2 by default). Ignores specified methods ({@code equals()} by default).
+ * (default = 2). Ignores specified methods ({@code equals()} by default).
```

Comment:
Maybe unrelated, but do we need the default listed in the description? We don't list default for other options and it seems to me we sometimes removed this in other cases.

Reply:
Let's remove them. I think the summaries should be short and clear. @romani @strkkk   what is your option?

Reply:
done